frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeaderView.tsx

```diff
@@ -120,52 +124,54 @@ export function DashboardHeaderComponent({
         isFixedWidth={dashboard?.width === "fixed"}
         isSidebarOpen={isSidebarOpen}
       >
-        <HeaderRow
-          className={cx("QueryBuilder-section", headerClassName)}
-          data-testid="dashboard-header"
-          ref={header}
-        >
-          <HeaderFixedWidthContainer
-            data-testid="fixed-width-dashboard-header"
-            isNavBarOpen={isNavBarOpen}
-            isFixedWidth={dashboard?.width === "fixed"}
+        {isDashboardHeaderVisible && (
```

Comment:
Only hide the part before the dashboard tabs, because the dashboard tabs should always be visible.